<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutProgrammers</class>
 <widget class="QWidget" name="AboutProgrammers">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>530</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>A B O U T     P R O G R A M M E R S</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(58, 37, 179);
</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ABOUT PROGRAMMERS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton#close:hover{
    background-color: rgb(99, 199, 147);
};

background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="person_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="setIcon.qrc">
     <normaloff>:/images/about/about-1.jpg</normaloff>:/images/about/about-1.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="person_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="setIcon.qrc">
     <normaloff>:/images/about/about-3.jpg</normaloff>:/images/about/about-3.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="person_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/about/about-2.png</normaloff>images/about/about-2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="person_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="setIcon.qrc">
     <normaloff>:/images/about/about-4.jpg</normaloff>:/images/about/about-4.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>140</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>name_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>name_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>name_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>name_4</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>email_1</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>email_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>email_3</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>email_4</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="setIcon.qrc"/>
 </resources>
 <connections/>
</ui>
